tests/generic_config_updater/test_monitor_config.py

```diff
@@ -0,0 +1,214 @@
+    expect_res_success(duthost, mirror_session, [
+        MONITOR_CONFIG_MIRROR_SESSION, MONITOR_CONFIG_POLICER], [])
+
+def verify_no_monitor_config(duthost):
+    """
+    Clean up monitor config in ACL_TABLE, ACL_RULE, POLICER, MIRROR_SESSION
+    """
+    table = duthost.shell("show acl table {}".format(MONITOR_CONFIG_ACL_TABLE))
+    expect_res_success(duthost, table, [], [MONITOR_CONFIG_ACL_TABLE])
+
+    rule = duthost.shell("show acl rule {} {}".format(MONITOR_CONFIG_ACL_TABLE, MONITOR_CONFIG_ACL_RULE))
+    expect_res_success(duthost, rule, [], [
+        MONITOR_CONFIG_ACL_TABLE, MONITOR_CONFIG_ACL_RULE, MONITOR_CONFIG_MIRROR_SESSION])
+
+    policer = duthost.shell("show policer {}".format(MONITOR_CONFIG_POLICER))
+    expect_res_success(duthost, policer, [], [MONITOR_CONFIG_POLICER])
+    
+    mirror_session = duthost.shell("show mirror_session {}".format(MONITOR_CONFIG_MIRROR_SESSION))
+    expect_res_success(duthost, mirror_session, [], [
+        MONITOR_CONFIG_MIRROR_SESSION, MONITOR_CONFIG_POLICER])
+
+def monitor_config_add_config(duthost, get_valid_acl_ports):
+    """ Test to add everflow always on config
+    """
+    json_patch = [
+        {
+            "op": "add",
+            "path": "/ACL_TABLE/{}".format(MONITOR_CONFIG_ACL_TABLE),
+            "value": {
+                "policy_desc": "{}".format(MONITOR_CONFIG_ACL_TABLE),
+                "ports": get_valid_acl_ports,
+                "stage": "ingress",
+                "type": "MIRROR_DSCP"
+            }
+        },
+        {
+            "op": "add",
+            "path": "/ACL_RULE",
+            "value": {
+                "{}|{}".format(MONITOR_CONFIG_ACL_TABLE, MONITOR_CONFIG_ACL_RULE): {
+                    "DSCP": "5",
+                    "MIRROR_INGRESS_ACTION": "{}".format(MONITOR_CONFIG_MIRROR_SESSION),
+                    "PRIORITY": "9999"
+                }
+            }
+        },
+        {
+            "op": "add",
+            "path": "/MIRROR_SESSION",
+            "value": {
+               "{}".format(MONITOR_CONFIG_MIRROR_SESSION): {
+                    "dscp": "5",
+                    "dst_ip": "2.2.2.2",
+                    "policer": "{}".format(MONITOR_CONFIG_POLICER),
+                    "src_ip": "1.1.1.1",
+                    "ttl": "32",
+                    "type": "ERSPAN"
+               }
+            }
+        },
+        {
+            "op": "add",
+            "path": "/POLICER",
+            "value": {
+                "{}".format(MONITOR_CONFIG_POLICER): {
+                    "meter_type": "bytes",
+                    "mode": "sr_tcm",
+                    "cir": "12500000",
+                    "cbs": "12500000",
+                    "red_packet_action": "drop"
+                }
+            }
+        }
+    ]
+
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    try:
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
```

Comment:
There is some dependency among ACL, mirror and policer. So, the order for applying different configs is required to respect the dependency. The correct order is supposed to be
```
Policer->mirror_session->ACL_TABLE->ACL_RULE
```

Reply:
The Yang schema will make sure the configs are added with the correct order.

Reply:
Let's assume the order for applying different configs is wrong. What happens to the orchagnt/syncd? Will they tolerate or complain, or crash?

Reply:
The config has its dependency. For example, ACL_RULE is based on ACL_TABLE, which means ACL_TABLE has to be applied first before ACL_RULE. The dependency is defined in yang(leafref specificly).
So if order is wrong, the `apply-patch` operation will fail and no further impact on orchagent/syncd.

Reply:
@bingwang-ms Could you check my question above?

Reply:
orchagent will complain some ERROR or WARNING logs if the referenced object is not found. It will not crash.